parser/parser.go

```diff
@@ -2138,8 +2170,10 @@ func (p *parser) parseLambdaExpr(allowCmd, allowRangeExpr bool) ast.Expr {
 			RhsHasParen: rhsHasParen,
 		}
 	}
-	if _, ok := x.(*tupleExpr); ok {
-		panic("TODO: tupleExpr")
+	if !allowTuple {
+		if _, ok := x.(*tupleExpr); ok {
+			p.error(x.Pos(), "not support tuple")
```

Comment:
`x.Pos()` => `tupleExpr.Opening`

Reply:
"not support tuple" => "tuple is not supported"

Reply:
fixed